File: curso-python/ex015.py
#quantidade de km percorrido
km = float(input("quantos km percorrido: ")) # 100km
dias = int(input("quantos dias o carro foi alugado: ")) # 4 dias
#calculo do preco
# carro custa R$60 por dia
# R$0.15 por km rodado
pago = (dias * 60) + (km * 0.15)
print(f"o total a pagar é de {pago:.2f}")

File: curso-python/javascript-exercicios/ex015.js
const prompt = require("prompt-sync")();
let km = Number.parseFloat(prompt("quantos km percorridos?: "))
let dias = Number.parseInt(prompt("quantos dias o carro foi alugado: "))

let pago = (dias * 60) + (km * 0.15)
console.log(`o total a pagar é de ${pago.toFixed(2)}`)